### PR TITLE
fix: Regenerate Azure resources

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Reruns the code generation with https://github.com/cloudquery/plugin-sdk/pull/112 included

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: plugins/source/azure/resources/services/storage/accounts.go [hunk 1]
```diff
@@ -168,7 +168,7 @@ func Accounts() *schema.Table {
 				Resolver: schema.PathResolver("AllowSharedKeyAccess"),
 			},
 			{
-				Name:     "enable_nfs_v_3",
+				Name:     "is_nfs_v_3_enabled",
```
Comment (disq) [inline plugins/source/azure/resources/services/storage/accounts.go:171]: `is_nfs_v_3_enabled`

**Comment by erezrokah** [comment]
@disq I think I addressed most of the things in https://github.com/cloudquery/cloudquery/pull/1754#issuecomment-1249745519. `ResourceURI` was already fixed and for `azure_compute_virtual_machine_scale_sets.host` and `azure_compute_virtual_machine_extensions` link to scale set, was this done in v1? I couldn't find a reference to it:
https://github.com/cloudquery/cq-provider-azure/blob/0b3011a0294187526f08a2ba9c68860a0716f83a/resources/services/compute/virtual_machine_scale_sets.go#L13

https://github.com/cloudquery/cq-provider-azure/blob/0b3011a0294187526f08a2ba9c68860a0716f83a/resources/services/compute/virtual_machines.go#L520

**Review by amanenk — APPROVED** [review]
LGTM

**Comment by erezrokah** [comment]
@disq If something is missing tag me and I'll do a follow up PR

**Pull request merged**